.github/workflows/tests.yaml

```
@@ -1,16 +1,15 @@
 name: Tests
 
 env:
-  DEFAULT_PYTHON_VERSION: "3.10"
   FORCE_COLOR: "1" # Make tools pretty.
+  PYTHONUNBUFFERED: "1" # Make stdout and stderr behave well
 
 on:
   pull_request:
   push:
     branches:
       - main
 
-
 # ensuring only one instance is running at a given time
 concurrency:
   group: ${{ github.ref }}
```

```suggestion
  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
```

[Explanation in `conda/conda`](https://github.com/conda/conda/blob/11c21d523f2e18e55b07479a31681cd4b63b8517/.github/workflows/tests.yml#L18-L25)